cmd/shfmt/main.go

```diff
@@ -67,9 +67,9 @@ func main1() int {
 	flag.Usage = func() {
 		fmt.Fprint(os.Stderr, `usage: shfmt [flags] [path ...]
 
-If no arguments are given, standard input will be used. If a given path
-is a directory, it will be recursively searched for shell files - both
-by filename extension and by shebang.
+If path is a single dash ('-') or absent, standard input will be used. If a
```

Comment:
> If the only argument is a dash ('-') or no arguments are given,

I don't want to imply that we support weird cases like `shfmt - -` or `shfmt - foo.sh`. Some of them we might, but I'd rather leave that undocumented, because it doesn't seem like a good idea to encourage.

Reply:
Done